### DISCUSSION TITLE
Проблема с восстановлением пароля

### DISCUSSION THREAD

**Discussion by XXXXPro** [OpenVK/openvk#301] [Q&A] [answered]
По каким-то причинам разлогинило несколько дней назад. Попытался восстановить пароль. Если ввожу свой id, то не получаю результата. Если ввожу Email, на который регистрировал профиль, получаю вот такое:

> Swift_TransportException
> 
> Failed to authenticate on SMTP server with username "notifications@openvk.su" using 3 possible authenticators. Authenticator LOGIN returned Expected response code 250 but got an empty response. Authenticator PLAIN returned Expected response code 250 but got an empty response. Authenticator XOAUTH2 returned Expected response code 250 but got an empty response.

> Call Stack:
> .../Swift/Transport/EsmtpTransport.php:371   Swift_Transport_Esmtp_AuthHandler->afterEhlo()
> .../Transport/AbstractSmtpTransport.php:148   Swift_Transport_EsmtpTransport->doHeloCommand()
> .../swiftmailer/lib/classes/Swift/Mailer.php:65   Swift_Transport_AbstractSmtpTransport->start()
> /opt/chandler/chandler/Email/Email.php:22   Swift_Mailer->send()
> 
> 12:            $transport->setUsername(CHANDLER_ROOT_CONF["email"]["addr"]);
> 13:            $transport->setPassword(CHANDLER_ROOT_CONF["email"]["pass"]);
> 14:            
> 15:            $message = new Swift_Message($subject);
> 16:            $message->getHeaders()->addTextHeader("Sensitivity", "Company-Confidential");
> 17:            $message->setFrom(CHANDLER_ROOT_CONF["email"]["addr"]);
> 18:            $message->setTo($to);
> 19:            $message->setBody($html, "text/html");
> 20:            
> 21:            $mailer = new Swift_Mailer($transport);
> 22:            return $mailer->send($message);
> 23:        }
> 24:    }
> 25:    
> .../chandler/MVC/SimplePresenter.php:161   Chandler\Email\Email::send()
> .../Web/Presenters/OpenVKPresenter.php:170   Chandler\MVC\SimplePresenter->sendmail()
> .../openvk/Web/Presenters/AuthPresenter.php:221   openvk\Web\Presenters\OpenVKPresenter->sendmail()
> .../chandler/chandler/MVC/Routing/Router.php:138   openvk\Web\Presenters\AuthPresenter->renderRestore()
> .../chandler/chandler/MVC/Routing/Router.php:172   Chandler\MVC\Routing\Router->delegateController()
> .../chandler/chandler/MVC/Routing/Router.php:289   Chandler\MVC\Routing\Router->delegateRoute()
> /opt/chandler/chandler/Bootstrap.php:120   Chandler\MVC\Routing\Router->execute()
> /opt/chandler/chandler/Bootstrap.php:147   Bootstrap->route()
> /opt/chandler/htdocs/index.php:7   Bootstrap->ignite()
> 

Судя по всему, при отправке письма с паролем не удаётся подключиться к почтовому серверу Mail.Ru либо из-за лимита подключений, либо из-за неправильного пароля в настройках.

**Comment by maksalees** [comment]
@XXXXPro Это проблема официального инстанса а не самого OpenVK

**Comment by ghost** [comment, marked as answer]
Исправлено.
